lens-driver-jdbc/src/main/java/org/apache/lens/driver/jdbc/JDBCDriver.java

```diff
@@ -735,7 +738,7 @@ private PreparedStatement prepareInternal(AbstractQueryContext pContext,
     // Only create a prepared statement and then close it
     MethodMetricsContext sqlRewriteGauge = MethodMetricsFactory.createMethodGauge(pContext.getDriverConf(this), true,
       metricCallStack + COLUMNAR_SQL_REWRITE_GAUGE);
-    String rewrittenQuery = rewriteQuery(pContext);
```

Comment:
Explain also works fine because of the below stack trace:
JDBCDriver.rewriteQuery(AbstractQueryContext) line: 529	
JDBCDriver.explain(AbstractQueryContext) line: 582	
QueryExecutionServiceImpl.explain(String, LensSessionHandle, String, LensConf) line: 3141	

Explain was also try to make JDBC connection, I have stopped this using JDBC_VALIDATE_THROUGH_PREPARE(I have renamed this variable to JDBC_VALIDATE_THROUGH_PREPARE_OR_EXPLAIN).